classifiers/reference_quality/chunked_sentence_complexity/code_snippet_common.md

```diff
@@ -0,0 +1,71 @@
+```python
+import textstat
+import spacy
+
+def get_mapping_complexity(score: int)-> str:    
+    if score < 30:
+        return "very difficult"
+    if score < 50:
+        return "difficult"
+    if score < 60:
+        return "fairly difficult"
+    if score < 70:
+        return "standard"
+    if score < 80:
+        return "fairly easy"
+    if score < 90:
+        return "easy"        
+    return "very easy"
+
+spacy_models_loaded = {}
+
+models = {
+    "en": "en_core_web_sm",
+    "de": "de_core_news_sm"
+}
+
+def get_spacy(language: str):
+    global spacy_models_loaded
+    model_name = models.get(language, "en_core_web_sm")
+    if model_name not in spacy_models_loaded:
+        spacy_models_loaded[model_name] = spacy.load(model_name)
+    return spacy_models_loaded[model_name]
+
+def chunked_sentence_complexity(text: str, language: str = "en") -> str:
+    """
+    @param text: 
+    @param language: iso language code
+    @spacy model: name of a language model from SpaCy 
```

Comment:
model is part of description but not part of function

Reply:
Re-added it